app/.yarnrc.yml

```diff
@@ -0,0 +1,19 @@
+yarnPath: .yarn/releases/yarn-3.2.4.cjs
```

Comment:
Why is `app/.yarn/releases/yarn-3.2.4.cjs` committed?

Reply:
Seems to be required. See <https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored>. Not sure if there's a way to avoid committing it.